packages/core/src/fhir-deduplication/__tests__/group-same-compositions.test.ts

```diff
@@ -0,0 +1,41 @@
+import { faker } from "@faker-js/faker";
+import { Composition, Extension } from "@medplum/fhirtypes";
+import { createFilePath } from "../../domain/filename";
+import { DOC_ID_EXTENSION_URL } from "../../external/fhir/shared/extensions/doc-id-extension";
+import { makeComposition } from "../../fhir-to-cda/cda-templates/components/__tests__/make-composition";
+import { groupSameCompositions } from "../resources/composition";
+
+let compositionId: string;
+let compositionId2: string;
+let composition: Composition;
+let composition2: Composition;
+let sourceExtension: Extension;
+
+beforeEach(() => {
+  compositionId = faker.string.uuid();
+  compositionId2 = faker.string.uuid();
+  sourceExtension = {
+    url: DOC_ID_EXTENSION_URL,
+    valueString: createFilePath(faker.string.uuid(), faker.string.uuid(), faker.string.uuid()),
+  };
+  composition = makeComposition(undefined, { id: compositionId, extension: [sourceExtension] });
+  composition2 = makeComposition(undefined, { id: compositionId2, extension: [sourceExtension] });
+});
+
+describe("groupSameCompositions", () => {
```

Comment:
We should prob also test that it won't group diff compositions, right? And that it works w/ an empty array of compositions